configs/encapsulate_http_in_http2_connect.yaml

```diff
@@ -0,0 +1,81 @@
+# This configuration takes incoming HTTP requests on port 10000 and encapsulates it in a CONNECT
+# request which is sent upstream port 10001.
+# It can be used to test HTTP tunneling as described in
+# https://envoyproxy.io/docs/envoy/latest/intro/arch_overview/http/upgrades
+# and running `curl -x 127.0.0.1:10000 https://www.google.com`
+
+admin:
```

Comment:
is admin required for this config? if not can we remove - i have tried to make the configs contain the least amount of configuration possible, esp when it comes to including admin

im also wondering the purpose of including these configs here - and whether this would be better served by adding a tunneling sandbox

Reply:
thinking on this further - why not just put these in the docs and add as literalinclude examples - it would be simpler than adding a sandbox, but at least they would be visible/used

Reply:
1. I'll remove admin, I thought it's required.
2. They are too large as literal include because they need the entirety to be meaningful. Snippets work best for isolated config samples.
3. Yes, users will copy paste so having a working example is really helpful.

Reply:
Let's just do both, add snippets and fully worked out examples.

Reply:
Resolving since we can have both snippets and full configs.